docs/constants/navigation.cjs

```diff
@@ -67,6 +67,7 @@ const general = [
   makeSection(
     'Conceptual Overview',
     [
+      makePage('introduction/expo.md'),
```

Comment:
You'll have to update the `navigation-deprecated.cjs` with the same change so that this page is shown in the navigation menu under Conceptual Overview.